pet006/sub-01/pet/sub-01_pet.json

```
@@ -0,0 +1,36 @@
+{
+	"AcquisitionMode": "list mode",
+	"AttenuationCorrection": "measured(emission present), 0.096000 cm-1, attenuation smooth",
+	"AttenuationCorrectionMethod": "measured(emission present), 0.096000 cm-1, attenuation smooth",
+	"BodyPart": "Phantom",
+	"DecayCorrectionFactor": [1.42614],
+	"DoseCalibrationFactor": 1323,
+	"FrameDuration": [98000],
+	"FrameTimesStart": [0],
+	"ImageDecayCorrected": true,
+	"ImageDecayCorrectionTime": 0,
+	"InjectedMass": 181149988.3691438,
+	"InjectedMassUnits": "ug",
+	"InjectedRadioactivity": 75.84999999999999,
+	"InjectedRadioactivityUnits": "MBq",
+	"InjectedVolume": 0,
+	"InjectionStart": 0,
+	"InstitutionName": "NIH Clinical Center, USA",
+	"Manufacturer": "GE MEDICAL SYSTEMS",
+	"ManufacturersModelName": "GE Advance",
+	"ModeOfAdministration": "infusion",
+	"ReconFilterType": "none",
+	"ReconMethodName": "3D Kinahan - Rogers",
+	"ReconMethodParameterLabels": [
+	  "none"
+	],
```

Okay, reading the spec, it says 

> REQUIRED if ReconMethodParameterLabels does not contain "none"

but that seems odd. If it's `["abc", "def", "none"]`, presumably you want units and values. I suppose the text should actually say 'is not `["none"]`'? I can probably do that in schema. Matching any might require me to bug Ross.

What is currently implemented is:

```suggestion
	"ReconMethodParameterLabels": "none",
```

We could update the text to allow that, but I think probably `["none"]` is the cleaner solution, so that the type is constant.

---

@mnoergaard Would you mind commenting on the original intent here?

---

The original intent goes back to the PET guidelines paper (https://journals.sagepub.com/doi/10.1177/0271678X20905433) where the ReconMethodParameterLabels was decided to be mandatory. In most cases, there are multiple reconstruction parameters and labels, however, for some older scanners there no recon parameters. As far as I remember, for the old validator, it was not easy to combine datatypes depending on the need, so it was decided to keep this as an array of strings. However, it is not so clean in the case of no parameters (even with ["none"]. Just "none" would be the cleanest.